include/inexor/vulkan-renderer/wrapper/device.hpp

```diff
@@ -80,6 +89,44 @@ class Device {
     [[nodiscard]] std::uint32_t get_transfer_queue_family_index() const {
         return transfer_queue_family_index;
     }
+
+    /// @brief Vulkan debug marker: Sets the name of a Vulkan resource.
+    /// The debug marker name will be visible in external debuggers like RenderDoc.
+    /// @param object [in] A pointer to the Vulkan object.
+    /// @param type [in] The type of the Vulkan object.
+    /// @param name [in] The name of the debug marker which will be associated to the Vulkan object.
+    void set_object_name(const std::uint64_t object, const VkDebugReportObjectTypeEXT type, const char *name);
+
+    /// @brief Vulkan debug marker: Links a memory dump block to a Vulkan resource.
+    /// The object will be visible in external debuggers like RenderDoc.
+    /// @param object [in] A pointer to the Vulkan object.
+    /// @param type [in] The type of the Vulkan object.
+    /// @param name [in] The name of the debug marker which will be associated to the Vulkan object.
+    /// @param tag_size [in] The size of the memory dump.
+    /// @param tag [in] A pointer to the memory dump.
+    void set_object_tag(const std::uint64_t object, const VkDebugReportObjectTypeEXT type, const std::uint64_t name,
+                        const std::size_t tag_size, const void *tag);
+
+    /// @brief Vulkan debug markers: Annotation of a rendering region.
+    /// The rendering region will be visible in external debuggers like RenderDoc.
+    /// @param command_buffer [in] The associated command buffer.
+    /// @param name [in] The name of the rendering region.
+    /// @param color [in] The rgba color of the rendering region.
+    void bind_region(const VkCommandBuffer command_buffer, const std::string &name, const float color[4]);
```

Comment:
Is there a way where we could pass `color` as an `std::array`?

Reply:
Sure.

Reply:
Done.